open-event-server/.env.old

```diff
@@ -1,12 +1,13 @@
-# DATABASE_URL=postgresql:///oevent
-DATABASE_URL=postgresql://open_event_user:opev_pass@localhost:5432/oevent
-# TEST_DATABASE_URL=postgresql:///opev_test
+DATABASE_URL=postgresql:///oevent
```

Comment:
Why do we have 2 .env files remaining?

Please at least describe in README on what each one is for, thanks.

Reply:
Thank you, @Sing-Li sorry for the silly mistake; I have updated the related docs. Please have a review.